src/Config/RuntimeConfigPath.cs

```diff
@@ -233,6 +233,11 @@ private static string GetOverriddenName(string fileName)
             return $"{fileName}.overrides{CONFIG_EXTENSION}";
         }
 
+        public static string GetEnvironmentFileName(string fileName, string environmentValue)
```

Comment:
in a method comment, can you note that fileName must not include an extension? i.e. not `myConfig.json`